src/runtime_src/core/tools/common/tests/TestIPU.h

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2023 Advanced Micro Devices, Inc. All rights reserved.
+
+#ifndef __TestIPU_h_
+#define __TestIPU_h_
+
+#include "tools/common/TestRunner.h"
+#include "xrt/xrt_bo.h"
```

Comment:
Doesn't look like bo is needed
```suggestion
#include "xrt/xrt_device.h"

#include <boost/property_tree/ptree.hpp>
#include <memory>
```